source/Outfit.cpp

```diff
@@ -476,7 +476,7 @@ int Outfit::CanAdd(const Outfit &other, int count) const
 
 		// Only automatons may have a "required crew" of 0.
 		if(!strcmp(at.first, "required crew"))
-			minimum = !attributes.Get("automaton");
+			minimum = !(attributes.Get("automaton") + other.attributes.Get("automaton"));
```

Comment:
Should this not be:
```suggestion
			minimum = !(attributes.Get("automaton") + count * other.attributes.Get("automaton"));
```
?

Reply:
as soon as one is there the count drops to zero, so no?

Reply:
```suggestion
			minimum = !(attributes.Get("automaton") || other.attributes.Get("automaton"));
```
although i think we could do this too

Reply:
The count here isn't necessary, since ShipCanSell always calls CanAdd with count = -1